Scuti/Scripts/UI/Account/VerifyEmailView.cs

```diff
@@ -32,6 +32,7 @@ protected override void Awake()
 
         public override void Open()
         {
+            UIManager.SetFirstSelected(firstSelection);
```

Comment:
@Jpablogs8 should the base Open method call SetFirstSelected so you don't have to change all of these files?

Reply:
Yes, this can be removed!